### ISSUE TITLE
Imports from celery.contrib could not be resolved

### ISSUE THREAD

**Issue by medihack** [sbdchd/celery-types#86]
When importing stuff from `celery.contrib` (e.g. `abortable.AbortableTask`) pyright tells me it could not be resolved. It seems to be missing in the stubs.

**Comment by sbdchd** [comment]
Ah yeah that's missing

When creating this project I basically only wrote types for the parts I needed -- probably should have used stub-gen https://mypy.readthedocs.io/en/stable/stubgen.html

**Comment by chatne** [comment]
Is there any way to get this added? Seems like alternative https://github.com/ryanwang520/celery-stubs includes contrib properly. However I would prefer this repository.

**Comment by vikigenius** [comment]
Yeah this still does not have it. I would love for this to get added

**Comment by 9en9i** [comment]
Are PR accepted? Can I add a small part that I need, instead of adding the entire module?
